antora.yml

```diff
@@ -33,6 +33,7 @@ asciidoc:
     tierthreeplan: Tiny Cloud Professional Plan
     enterpriseplan: Tiny Custom Plans
     ## common tiny.cloud URLs
+    url: https://www.tiny.cloud
```

Comment:
This is too generically named... it probably needs to be called `siteurl` or similar to make it clear what the URL is.

Reply:
Yeah the thing is this attribute is already used in the docs but it was just not defined in the antora.yml file, so do I change all occurrences of this variable to `siteurl` or do I just leave it as it was before ?

Reply:
as a reference and for consistency, the 6x docs uses `companyurl`

see https://github.com/tinymce/tinymce-docs/blob/release/docs-6/antora.yml#L6